example/Docs: Removed env from collector-k8-manifest & updated tutori… (#11)
kubearmor/otel-adapter: example/tutorials/tutorial.md

- `config.yml` file is located in this repo at `example/config.yml`.
Examine the logs to see that it is properly running.

***Learn about the receiver's configuration options [here](tutorial.md#kubearmor-receiver-config).***

#### Cleanup
```bash
# stop and remove the collector container
docker stop kubearmor-otel-receiver; docker rm kubearmor-otel-receiver
```

### Kubearmor receiver config.

There are two configuration options for the receiver:

- **endpoint:**
  

>    This specifies kubearmor's server API URL.

Reviewer: ```suggestion
   This specifies kubearmor's log server's gRPC endpoint.
```

  
- **logfilter**
  
   This is used to specify which logs one is interested in. There are three filters:
  
   - kubearmorLogs:
     

>      Use this if you want to see Kubearmor's application logs only.

Reviewer: ```suggestion
     Use this if you want to see Kubearmor's internal logs only.
```

     
   - policy
     
     Use this if you want to see alerts only.
  
   - system
     
     Use this if you want to see logs about insights gotten by kubearmor about the host system only.
     
   - all
     

>       Use this if you want to see the entire log messages only.

Reviewer: ```suggestion
      Use this if you want to see all the alerts, visibility and internal logs.
```

docker stop kubearmor-otel-receiver; docker rm kubearmor-otel-receiver
```

> ### Kubearmor receiver config.

Reviewer: Let's also refer this in the step where we create the deployments -
https://github.com/Chinwendu20/OTel-receiver/blob/e8b8f5b53b8fd3aec9435b850d1c0a0433549d08/example/tutorials/tutorial.md?plain=1#L34
https://github.com/Chinwendu20/OTel-receiver/blob/e8b8f5b53b8fd3aec9435b850d1c0a0433549d08/example/tutorials/tutorial.md?plain=1#L75

Reviewer: Also, the preview doesn't look as intended :sweat_smile: - https://github.com/Chinwendu20/OTel-receiver/blob/env/example/tutorials/tutorial.md#kubearmor-receiver-config
